tomls/omnibus-base-sepolia-andromeda/perps/wif-invokes.toml

```diff
@@ -0,0 +1,147 @@
+include = [
+    # TODO: this is a funny necessary dependency because the oracle manager is registering the same node twice
+    # however, it will not emit "NodeRegistered" the second time because its an unnecessary call
+    "../../oracles/pyth-wif.toml",
+    "../../markets/common/bigcap-settings.toml",
+]
+# NOTE: This toml will live until the setMaxMarketValue is on mainnet
+
+[setting.perpsWifMarketId]
+
+[setting.perpsWifSkewScale]
+
+[setting.perpsWifMaxFundingVelocity]
+
+[setting.perpsWifMakerFeeRatio]
+
+[setting.perpsWifTakerFeeRatio]
+
+[setting.perpsWifMaxMarketSize]
+
+[setting.perpsWifInitialMarginRatio]
+
+[setting.perpsWifMaintenanceMarginFraction]
+
+[setting.perpsWifFlagRewardRatioD18]
+
+[setting.perpsWifMaxLiquidationLimitAccumulationMultiplier]
+
+[setting.perpsWifMaxSecondsInLiquidationWindow]
+
+[setting.perpsWifMaxLiquidationPd]
+
+[setting.perpsWifEndorsedLiquidator]
+
+[setting.perpsWifMinimumPositionMargin]
+
+[setting.perpsWifLockedOiRatio]
+
+[invoke.createWifPerpsMarket]
+target = ["perpsFactory.PerpsMarketProxy"]
+fromCall.func = "owner"
+func = "createMarket"
+args = ["<%= settings.perpsWifMarketId %>", "Wif", "WIF"]
+
+[invoke.setPerpsPriceWif]
+target = ["perpsFactory.PerpsMarketProxy"]
+fromCall.func = "owner"
+func = "updatePriceData"
+args = [
+    "<%= settings.perpsWifMarketId %>",
```

Comment:
Can we export market id with extras, use it here instead of hardcoded setting and then remove depends? On all steps